src/settings/PatronNotices/PatronNoticeForm.js

```diff
@@ -256,12 +252,14 @@ class PatronNoticeForm extends React.Component {
                 data-test-patron-notice-template-name
               >
                 <Field
-                  required
-                  label={<FormattedMessage id="ui-circulation.settings.patronNotices.notice.name" />}
+                  label={
```

Comment:
It might be a good idea to use "required" instead of adding an asterisk manually, what do you think about it?

Reply:
No, this attribute will broke our form validation, and also all required fields in circulation module follows the same pattern.